### ISSUE TITLE
Updated

### ISSUE THREAD

**Issue by juzman** [seud0nym/tch-gui-unhide#13]
Not so much an "issue" persay, but just writing to report that I upgraded to 18.1.c.0585-RA on my DJA0231 and re-ran de-telstra, disabled dumaos and ran tch-gui-unhide and all is working beautifully after a clean flash. :)

**Comment by seud0nym** [comment]
Good news. Do you have a download link, so I can check for changes?

**Comment by juzman** [comment]
Still have the red status light on front, even though WAN/DSL and ONLINE are green on back. Only thing I can think of is I disabled all the telephony stuff so maybe the modem thinks it's not "fully working". Ah well, aesthetics. Haha.

Other thing I noticed with 585 is dumaos is enabled by default, so have to run dumaos -off 👍 

http://fwstore.bdms.telstra.net/Technicolor_vcnt-a_18.1.c.0585-MR7.1-RA.rbi/vcnt-a_18.1.c.0585-MR7.1-RA.rbi

**Comment by seud0nym** [comment]
Thanks for that. How do you get them so early?

Interestingly, I have just installed as a clean flash on my test DJA0231, and dumaos is not enabled (I haven't run de-telstra or tch-gui-unhide yet). I wonder what the difference could be?

**Comment by juzman** [comment]
That's strange. No idea why dumaos was enabled by default.

I may or may not have access to the CMS servers which push out firmware
updates, so periodically check when new ones are out. ;)

**Comment by seud0nym** [comment]
I understand :D 

I appreciate you letting me know when a new update is available.

**Comment by juzman** [comment]
No worries, appreciate the work you put into this software.

One day if like to create a "light" version with necessities (VDSL2, WAN,
4G, and WiFi).

Completely remove the TOD, Parental Controls, File Sharing, UPNP etc.
Basically everything I disable anyway.

On another note, rebooted the modem 3 times and the light on the front is
green now, instead of red but I didn't change anything. Haha.

**Comment by seud0nym** [comment]
My production DJA0231 did the red light thing the other day. I disabled/enabled the connection a few times and it came back. Weird.

I have pushed up a new version of de-telstra that has a new -M option for minimal memory configuration, which is equivalent to specifying n on all u/y/n options (except power saving and WPS), plus all services associated with those options are stopped and disabled.

I have also pushed up a new version of tch-gui-unhide-cards that has a new option to hide all cards for optional features.

I will be doing a release probably later today.
